webrender/src/renderer.rs

```diff
@@ -2252,7 +2254,7 @@ impl<B: hal::Backend> Renderer<B> {
                         &mut profile_timers,
                         //&profile_samplers,
```

Comment:
Exactly, because `profile_samplers` uses code which is strongly OpenGL dependent, namely the `query` modul, which we [don't use](https://github.com/szeged/webrender/blob/master/webrender/src/lib.rs#L95) for the previously mentioned reason. I think this is more relevant to the `GpuProfiler` part of #150.